src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -312,6 +331,16 @@ protected override Encoding StandardOutputEncoding
 
         protected override int ExecuteTool(string pathToTool, string responseFileCommands, string commandLineCommands)
```

Comment:
The fix for that issue is to add a /pdb: option to the VB command-line and remove the special-case in Vbc task. See #3859.